R/pmcmc.R

```diff
@@ -61,13 +66,32 @@
 ##'
 ##' @export
 pmcmc <- function(pars, filter, n_steps, save_state = TRUE,
-                  save_trajectories = FALSE, progress = FALSE) {
+                  save_trajectories = FALSE, progress = FALSE,
```

Comment:
Add the burn-in option here? Or note that it can be thinned/sampled from later.
And also just to check – can you still remove burn in after the chains have been combined, or does it look like one big chain?

Reply:
burnin/thin is purposefully done later, and it works as you'd want on a combined chain (I think you reviewed this the other day!)

Reply:
haha, sorry. I'd forgotten whether they get put into a single row or into columns - but yes I do recall that you keep the number of chains